### ISSUE TITLE
Discontinued 

### ISSUE THREAD

**Issue by DevGod6969** [ArchonKlaus/GTA#4]
Is  this Project discontinued or something please update it

**Comment by samupozzo** [comment]
Yes, the project is actually discontinued **due to massive api changes**.
